src/components/radio/StyledRadio.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from 'emotion-solid'
+import tw from 'twin.macro'
+
+import { IRadioProps } from './Radio'
+
```

Comment:
Maybe move the `IRadioProps & JSX.IntrinsicElements['label']` up here to it's own type:
```
type StyledRadioProps = RadioProps & JSX.IntrinsicElements['label']
```

Reply:
Yep, sounds good